app/routes/o.$orgSlug/index.tsx

```diff
@@ -1,9 +1,43 @@
+import { convexQuery } from "@convex-dev/react-query";
 import { createFileRoute } from "@tanstack/react-router";
+import { getCurrentUser } from "@/app/functions/auth";
+import TestonomialForm from "@/components/forms/testinomial-form";
+import Navbar from "@/components/navbar";
+import { api } from "@/convex/_generated/api";
 
 export const Route = createFileRoute("/o/$orgSlug/")({
-  component: RouteComponent,
+  component: TestimonialSubmissionPage,
+  loader: async ({ context: { queryClient }, params }) => {
+    const user = await getCurrentUser();
+    const organization = await queryClient.ensureQueryData(
+      convexQuery(api.organization.getOrganizationBySlug, {
+        slug: params.orgSlug,
+      }),
+    );
```

Comment:
No need to fetch the organization again. I think it's already defined in the `context` object?